docs/go/versioning.md

```diff
@@ -10,7 +10,7 @@ a non-deterministic issue if not handled correctly.
 
 ## Introduction to Versioning
 
-Because we design for potentially long running workflows at scale, versioning with Temporal works differently than with other workflow systems. We explain more in this optional 30 minute introduction:
+Because we design for potentially long running Workflows at scale, versioning with Temporal works differently than with other Workflow systems. We explain more in this optional 30 minute introduction:
```

Comment:
```suggestion
Because we design for potentially long running Workflows at scale, versioning with Temporal works differently. We explain more in this optional 30 minute introduction:
```